ios/app/watchos/extension/MeetingRowController.swift

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright @ 2018-present 8x8, Inc.
+ * Copyright @ 2017-2018 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import WatchKit
+
+class MeetingRowController: NSObject {
```

Comment:
I wonder why this inherits from NSObject (or if that's needed, actually)

Reply:
I've just checked and without it the app crashes. Something about selectors. It's this way in the docs too:
https://developer.apple.com/documentation/watchkit/wkinterfacetable

Reply:
```2019-05-01 12:20:37.964413-0500 JitsiMeetCompanion Extension[334:234001] *** NSForwarding: warning: object 0x15670690 of class 'JitsiMeetCompanion_Extension.MeetingRowController' does not implement methodSignatureForSelector: -- trouble ahead
Unrecognized selector -[JitsiMeetCompanion_Extension.MeetingRowController init]
2019-05-01 12:20:37.964907-0500 JitsiMeetCompanion Extension[334:234001] Unrecognized selector -[JitsiMeetCompanion_Extension.MeetingRowController init]
warning: could not execute support code to read Objective-C class data in the process. This may reduce the quality of type information available.```